software/mainsail.yml

```
@@ -0,0 +1,13 @@
+name: Mainsail
+website_url: https://docs.mainsail.xyz/
+description: A modern and responsive user interface for Klipper. Control and monitor your printer from everywhere, from any device.
+licenses:
+  - GPL-3.0
+platforms:
+  - TypeScript
```

```suggestion
  - Docker
  - Python
```

> Values for platform are the main server-side requirements for the software. Don't include frameworks or specific dialects.

- https://docs.mainsail.xyz/setup/getting-started/docker
- https://docs.mainsail.xyz/setup/getting-started/manual-setup